templates/collections/results_page.html

```diff
@@ -14,7 +14,7 @@
                     <h2 class="sr-only">Results</h2>
                     <h3 class="sr-only">Options</h3>
                     {% include 'includes/results-options.html' %}
-                    <ul class="row card-group">
+                    <ul class="row card-group--list-style-none">
```

Comment:
Similar to comments above, we should avoid mixing BEM and Bootstrap on the same element.